### PR TITLE
Implement logging

### PR DESCRIPTION
**Pull request by ChenJesse**
Log looks like
```
2017-11-12 14:06; INFO; (follower_id: 7193, followed_id: 7194) following created
2017-11-12 14:06; INFO; (follower_id: 7193, followed_id: 7195) following created
2017-11-12 14:06; INFO; (follower_id: 7196, followed_id: 7197) following created
2017-11-12 14:06; INFO; (follower_id: 7196, followed_id: 7197) following deleted
2017-11-12 14:06; INFO; (follower_id: 7199, followed_id: 7200) following created
2017-11-12 14:06; INFO; (follower_id: 7205, followed_id: 7206) following created
2017-11-12 14:06; INFO; (follower_id: 7205, followed_id: 7206) following deleted
2017-11-12 14:06; INFO; (follower_id: 7208, followed_id: 7209) following created
2017-11-12 14:06; INFO; (follower_id: 7208, followed_id: 7210) following created
2017-11-12 14:06; INFO; (follower_id: 7211, followed_id: 7212) following created
2017-11-12 14:06; INFO; (follower_id: 7211, followed_id: 7212) following deleted
2017-11-12 14:06; INFO; (follower_id: 7214, followed_id: 7215) following created
2017-11-12 14:06; INFO; (follower_id: 7220, followed_id: 7221) following created
2017-11-12 14:06; INFO; (follower_id: 7220, followed_id: 7221) following deleted
```

They are stored in `podcast-backend/src/app/logs`, and are named with the year-month-day, like `info-2017-Nov-12.log`. Configured to rotate through 5 backup files so it doesn't become just 1 massive log

### CODE REVIEW

**Comment by DanielNosrati** [comment]
Do we then need to modify https://github.com/cuappdev/appdev.py/blob/master/appdev/controllers/app_dev_controller.py since this is where all exceptions get caught? Or will we not care about those exceptions?

**Comment by ChenJesse** [comment]
Yeah I was thinking we don't really care about those exceptions, not sure if we should care or not, guess that's up for discussion. I feel like most of the important exceptions were the ones we were printing for (db_utils ones and some session creating ones)

File: src/app/pcasts/controllers/google_sign_in_controller.py [hunk 1]
```diff
@@ -18,6 +18,10 @@ def content(self, **kwargs):
     user, is_new_user = \
         users_dao.get_or_create_user_from_google_creds(google_user_info)
     session = sessions_dao.get_or_create_session_and_activate(user.id)
+    app.logger.info(
+        '(id: %s, username: %s, platform: %s) session created successfully',
+        user.id, user.username, 'google'
```
Comment (DanielNosrati) [inline src/app/pcasts/controllers/google_sign_in_controller.py:23]: Our constants file now/will include the string for google/facebook

**Review by DanielNosrati — APPROVED** [review]
Search itunes probably also needs one. It seems good to see if someone abuses our searches especially since it manipulates our data

**Pull request merged**